pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/generate/BaseGenerator.java

```diff
@@ -76,13 +77,13 @@ public Object collect() {
         Object sourceToUpsert;
         try {
             sourceToUpsert = parseJson();
-        } catch (JsonParseException jpe) {
+        } catch (JsonParseException | DsmNullDateException e) {
```

Comment:
In case if `DsmNullDateException` we also need to parse the nameValue as single element